backend/bitboxbase/bitboxbase.go

```diff
@@ -74,6 +74,38 @@ type Interface interface {
 
 	// SyncOption returns true if the bitcoin resync rpc call executed successfully
 	SyncWithOption(SyncOption) (bool, error)
+
+	// GetHostname returns the hostname of the BitBox Base
+	GetHostname() (string, error)
+
+	// SetHostname sets the hostname of the BitBox Base
+	SetHostname(string) (bool, error)
```

Comment:
This applies to all the getters that return `(bool, error)`
With how we designed our error handling we can always assume success if there is no error so the success bool is redundant right? So we could only have an `error` return type everywhere and `return nil` when successful. 
Then in the handlers you don't have to assign two variables with `success, err := handlers.base...` and can just return `return map[string]interface{}{"success": true}, nil` instead of `return map[string]interface{}{"success": success}, nil`